.circleci/config.yml

```diff
@@ -151,6 +157,20 @@ jobs:
     steps:
       - run-e2e-tests-for-path:
           path: "./spec/e2e/features/short_form/!(submitting_application|sign_in_while_applying).feature"
+  e2e-production:
+    <<: *defaults
+    steps:
+      - setup
+      - start-server
+      - run-e2e-tests-for-path:
```

Comment:
is the idea to run prod tests all serially? or just waiting til marcin's pr to parallelize

Reply:
@akegan You know, this was just out of wanting to keep the config simple. I wasn't thinking at all about serial vs parallel. I think it would be better to make these parallel, changing it now!